src/Isa/Acc32.hs

```diff
@@ -26,7 +26,7 @@ instance Hashable Register
 -- | The 'Isa' type represents the instruction set architecture for the Acc32 machine.
 -- Each constructor corresponds to a specific instruction.
 data Isa w l
-    = -- | Syntax: @load_imm <value>@ Load an immediate value into the accumulator.
+    = -- | Syntax: @load_imm <address>@ Load an immediate value into the accumulator.
```

Comment:
Yeah, I'm a sure about that. There is usage example:
   load_imm   buf
   
 Also in the code for load_imm we have: 
   LoadImm l -> LoadImm (deref' f l)
   
  And the command realisation:
  LoadImm a -> setAcc a >> nextPc
 
So we understand that the command receives the address of the label and loads this address value into the acc, but we can't directly load a value, because of the "deref' f l" instruction. 

Maybe it should be like that:
Syntax: @load_imm <address>@ Load an immediate value _**of the address**_ into the accumulator.